prow/plugins/trigger/ic.go

```
@@ -54,20 +54,23 @@ func handleIC(c client, trigger *plugins.Trigger, ic github.IssueCommentEvent) e
 	}
 
 	// Which jobs does the comment want us to run?
-	okToTest := okToTestRe.MatchString(ic.Comment.Body)
-	testAll := okToTest || testAllRe.MatchString(ic.Comment.Body)
+	isOkToTest := okToTestRe.MatchString(ic.Comment.Body)
+	testAll := isOkToTest || testAllRe.MatchString(ic.Comment.Body)
 	shouldRetestFailed := retestRe.MatchString(ic.Comment.Body)
 	requestedJobs := c.Config.MatchingPresubmits(ic.Repo.FullName, ic.Comment.Body, testAll)
 	if !shouldRetestFailed && len(requestedJobs) == 0 {
 		// Check for the presence of the needs-ok-to-test label and remove it
 		// if a trusted member has commented "/ok-to-test".
-		if okToTest && ic.Issue.HasLabel(NeedsOkToTest) {
+		if isOkToTest && ic.Issue.HasLabel(NeedsOkToTest) {
 			trusted, err := trustedUser(c.GitHubClient, trigger, commentAuthor, org, repo)
 			if err != nil {
 				return err
 			}
 			if trusted {
-				return c.GitHubClient.RemoveLabel(org, repo, number, NeedsOkToTest)
+				if err := c.GitHubClient.RemoveLabel(org, repo, number, NeedsOkToTest); err != nil {
+					c.Logger.WithError(err).Errorf("Failed at removing %s label", NeedsOkToTest)
```

Let's just return the error here rather than continuing on (same deal below)